HELM-341: Documentation formatting cleanup (#484)
OpenNMS/opennms-helm: docs/modules/getting_started/pages/importing.adoc

[.lead]

[[gs-dashboard-import]]

> = Import A Dashboard

Reviewer: ```suggestion
= Import a Dashboard
```
Should be lower case in this context.

* Network interfaces
* And more ...

> You can edit these pre-designed dashboards to meet your needs or to learn more about how to build your own dashboards.

Reviewer: ```suggestion
You can edit these predesigned dashboards to meet your needs or to learn more about how to build your own dashboards.
```

* Network interfaces
* And more ...

You can edit these predesigned dashboards to meet your needs or to learn more about how to build your own dashboards.

To import an existing dashboard, follow these steps:

. Go to the https://grafana.com/grafana/dashboards[Grafana dashboards page] and search for OpenNMS.
. Click on the dashboard you want to import.
. Copy the dashboard ID:
+
image:gf-community-dash.png[]

. In {page-component-title}, click the plus sign (*+*) in the left menu and choose *Import*.
. Type the dashboard ID in the *Import via grafana.com* field and click *Load*.
+
The Import screen appears.

. Change information as desired and click *Import*.
+
You must specify a flow and performance data source.
+
The dashboard appears.
+

> . Click *Dashboards > Manage* to view a list of your dashboards.

Reviewer 2: ```suggestion
. Click menu:Dashboards[Manage] to view a list of your dashboards.
```
